UPGRADE.md

```diff
@@ -0,0 +1,25 @@
+# Network Upgrade
+
+1. Wait for the proposed upgrade block. Your node will panic and stop once the block is reached. If you are running a validator, you will need to vote for the upgrade proposal beforehand. You can do so by running
+```
+axelard tx gov vote ${proposal_id} yes \
+	--from validator --gas auto --gas-adjustment 1.5
+```
+
+2. Backup the state and keys.
+```
+mv /home/axelard/.axelar /home/axelard/.axelar_backup
```

Comment:
```suggestion
mv $ROOT_DIRECTORY/.axelar $ROOT_DIRECTORY/.axelar_backup # run from outside the container
```

Reply:
Are people supposed to have a `ROOT_DIRECTORY` env var outside the container?  I don't have one on my node.